src/backend/distributed/shared_library_init.c

```diff
@@ -379,6 +380,19 @@ RegisterCitusConfigVariables(void)
 		GUC_NO_SHOW_ALL,
 		NULL, NULL, NULL);
 
+	DefineCustomBoolVariable(
+		"citus.enable_replace_table_visible_function",
```

Comment:
did not like the name here. `enable` seem to be redundant. `override_table_visibility` or `use _citus_table_visibility` sounds better to me